treeherder/services/pulse/consumers.py

```diff
@@ -162,6 +163,36 @@ def on_message(self, body, message):
         )
         message.ack()
 
+class Pulses(PulseConsumer):
+    queue_suffix = env("PULSE_SOMETHING", default="need")
+
+    def bindings(self):
+        rv = []
+        if self.source.get('hgmo'):
+            rv += HGMO_PUSH_BINDINGS
+        elif self.source.get('github'):
+            rv += GITHUB_PUSH_BINDINGS
+        else:
+            return TASKCLUSTER_TASK_BINDINGS 
+        return rv
+
+    @newrelic.agent.background_task(name='pulse-listener-tasks.on_message', group='Pulse Listener')
```

Comment:
Where do I find this decorator function?

Reply:
I believe I mentioned in the answer in Matrix.

Please adjust the value of `name`.